<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRenameDialog</class>
 <widget class="QWidget" name="AddRenameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#AddRenameDialog #rounded_wdg
{
	background-color:  white;
	color: black;
	border-radius: 8;
	border: 1px solid lightgrey;
}


#AddRenameDialog #titleLbl
{
	color: black;
}



/*
*				X button
*
*/

#AddRenameDialog #closeEditorBtn
{
	background-color: transparent;
	border-radius: 8;
	border: 0px solid transparent;
}


#AddRenameDialog #closeEditorBtn:hover
{
	background-color: rgb( 255, 235, 235 );
}

#AddRenameDialog #closeEditorBtn::pressed
{
	background-color: rgb( 240, 210, 210 );
}



#AddRenameDialog #top_lbl,
#AddRenameDialog #bottom_lbl
{
	color: black;
}


/*
*				LINE EDIT
*
*/
#AddRenameDialog #top_le,
#AddRenameDialog #bottom_le
{
	padding-top: 2;
	padding-bottom: 2;
	padding-left: 4;

	border-radius: 4;
	border: 1px solid black;
	background-color: white;
	color: black;
}

#AddRenameDialog #top_le:focus,
#AddRenameDialog #bottom_le:focus
{
	border: 1px solid rgb( 120, 120, 245 );;
}


/*
*				PUSHBUTTON
*
*/
#AddRenameDialog #submit_btn
{
	border-radius: 8;
	color: white;
	background-color:  rgb( 128, 128, 255 );
	border: 2px solid rgb( 170, 170, 255 );
}


#AddRenameDialog #submit_btn:hover
{
	background-color:  rgb( 158, 158, 255 );
	border: 2px solid rgb( 170, 170, 255 );
}



#AddRenameDialog #submit_btn::pressed
{
	background-color:  rgb( 118, 118, 255 );
	border: 2px solid rgb( 150, 150, 205 );
}



</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="rounded_wdg" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="titleLbl">
          <property name="font">
           <font>
            <family>Inter</family>
            <pointsize>24</pointsize>
            <fontweight>DemiBold</fontweight>
           </font>
          </property>
          <property name="text">
           <string>Group Editor</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="closeEditorBtn">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../assets.qrc">
            <normaloff>:/icons/topLvl/asset/icons/X_black.svg</normaloff>:/icons/topLvl/asset/icons/X_black.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::Shape::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="sp_3">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="top_lbl">
        <property name="font">
         <font>
          <family>Inter</family>
          <pointsize>16</pointsize>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="text">
         <string>GroupName:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="top_le">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Inter</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="sp_1">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="bottom_lbl">
        <property name="font">
         <font>
          <family>Inter</family>
          <pointsize>16</pointsize>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="text">
         <string>Rename Group to:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="bottom_le">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Inter</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="sp_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="submit_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>48</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Inter</family>
          <pointsize>16</pointsize>
          <fontweight>Medium</fontweight>
         </font>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../assets.qrc"/>
 </resources>
 <connections/>
</ui>
